### DISCUSSION TITLE
"Helpful Macros" - Where did they go?

### DISCUSSION THREAD

**Discussion by Jeppedy** [Guilouz/Creality-Helper-Script-Wiki#684] [Issues]
In the documentation, it refers to an Option 11 that installs "Helpful Macros" like PID Tuning.  But I don't see them anymore in v6.2.0.  Not a big deal, but just asking.

Thanks for a great tool.  The more I work with my Creality 3 v3 KE, the more I realize it's a unique beast and needs a Creality Helper Script to do things that work in other setups.  After working with it for a while, I just may start looking to see if I can install from bare metal to have a more standard setup.  But until then, I have your script!!

**Comment by Jeppedy** [comment]
No one knows?

**Reply by mfraser** [reply]
According to the menu script for the 3V3 it is:
`menu_option ' 9' 'Install' 'Useful Macros'`
